firefox_ui_tests/remote/security/test_security_notification.py

```diff
@@ -0,0 +1,61 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+
+from marionette import By
+from marionette.errors import MarionetteException
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestSecurityNotification(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.urls = [
+            # Invalid cert page
+            'https://summitbook.mozilla.org',
+            # Secure page
+            'https://ssl-ev.mozqa.com/',
+            # Unsecure page
+            'http://www.mozqa.com'
+        ]
+
+        self.identity_box = self.browser.navbar.locationbar.identity_popup.box
+
+    def test_security_notification(self):
+        # Go to a secure (https) site
+        with self.marionette.using_context('content'):
+            self.marionette.navigate(self.urls[1])
+
+        self.wait_for_condition(lambda _: self.identity_box.get_attribute('className') ==
+                                'verifiedIdentity')
+
+        # Go to an insecure (http) site
+        with self.marionette.using_context('content'):
+            self.marionette.navigate(self.urls[2])
+
+        self.wait_for_condition(lambda _: self.identity_box.get_attribute('className') ==
+                                'unknownIdentity')
+
+        self.marionette.set_context('content')
```

Comment:
To be safe you should use `with self.marionette.set_context('content')` here. Otherwise we stay in content for teardown if anything below fails.